lte/gateway/c/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -331,7 +331,7 @@ int amf_proc_security_mode_control(
   int amf_eea = 0;
   int amf_eia = 0;  // Integrity Algorithm 2
   amf_plmn_t plmn;
-  uint8_t snni[32]  = {0};
+  uint8_t snni[40]  = {0};

```

Comment:
no need to change to 40 as this is 32 bytes.

Reply:
reverted